common/templates/includes/map.njk

```diff
@@ -0,0 +1,220 @@
+<div class="govuk-grid-column-full" style="height: 600px; position: relative">
+  <div id="map" class="map" class="govuk-grid-column-full" style="height: 600px"></div>
+  <div id="message" class="govuk-grid-column-full govuk-heading-l" style="position: absolute; top: 100px; height: 200px; text-align: center"></div>
+</div>
+
+{#  <script src="https://cdnjs.cloudflare.com/ajax/libs/openlayers/4.6.5/ol.js" integrity="sha512-O7kHS9ooekX8EveiC94z9xSvD/4xt10Qigl6uEKvspYykdux3Ci5QNu5fwi4ca0ZkZI/oCgx5ja8RklWUEqzxQ==" crossorigin="anonymous"></script>#}
+{#  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/openlayers/4.6.5/ol.css" integrity="sha512-D2c/nmSVlwoZk29SLwJK8GkLyCyPuNbZ/vZQL/lNzw+o+yXzsbnJt7lK/iCfdBmgyLRDi0qx27gpE8Zb1C6I7g==" crossorigin="anonymous" />#}
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/openlayers/4.6.5/ol-debug.css" integrity="sha512-hBSieZLd5rse9gdkfv4n0pDU4D04SxpqBtwDzRy/QiXRBhczDyfCTDTnHCada73ubNqiQv6BLgCRXHAJPUwC5w==" crossorigin="anonymous" />
+<script src="https://cdnjs.cloudflare.com/ajax/libs/openlayers/4.6.5/ol-debug.js" integrity="sha512-RGRzp3Gfp9l+vkKXNjm45LxXEujXIZBIur2l2CRqIYDiZ+QLe2hprjK7YZBKo1RrfbF/pBvg2431gHU7pjjJ1A==" crossorigin="anonymous"></script>
```

Comment:
Do we know if the people using this will be on some kind of restricted network? Like Quantum, etc? I am unsure if Quantum will allow a connection to cloudflare.

Reply:
Urgh, that's a good point - I'd forgotten about restricted networks.

As I said in the description, I've done it this way so that we don't have to immediately implement Webpack [Code Splitting](https://webpack.js.org/guides/code-splitting/) and [Lazy Loading](https://webpack.js.org/guides/lazy-loading/) as that can be a bit of a mine field for development.

@martin-ballhatchet-moj Thoughts on the above?

Reply:
Hi. So - I'm fine with Cloudflare for loading static assets like this. I would hope that Quantum doesn't block Cloudflare domains. It would be good to know what the user would see if that is blocked though - to help troubleshooting later.
I'm a little worried that we may deploy this to "production" with the debug versions of the CSS and JS - would be good to have a way to prevent that (even if it's a JIRA ticket to switch them over!)

Reply:
@martin-ballhatchet-moj re: `debug` - That's a good point - I'll flip around the prod/debug includes for this PR. 

I'll still keep the debug lines, but commented out, as it's not immediately obvious how to get those URLS - we should only be needing debug on local machines anyway.